### PR TITLE
Add Scala Bridge, Lambda World, an article and a talk

### PR DESCRIPTION
**Pull request by jorgegalindocruces**
Hi everyone!

With this PR I propose:
- Scala Bridge organization
- Lambda World conference
- A post of Laura Bell that personally I think a must to every event organizer
- A talk by Eugenia Cheng to explain Category Theory speaking about privileges in the real life

Cheers and thank you, folks, for this!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +118,7 @@ Talks and presentations about diversity, consensual software and safe communitie
 
 - [9 Ways to Stop Hurting and Start Helping Women in Tech](https://www.youtube.com/watch?v=pUVhF3jDG08) - By Brianna Wu.
 - [Consensual Software: Prioritizing Trust & Safety for new features](https://www.youtube.com/watch?v=Ccw3VfE3P4M) - By Danielle Leong.
+- [Cathegory Theory in Life](https://youtu.be/ho7oagHeqNc) - By Eugenia Cheng
```
Comment (dreamingechoes) [inline README.md]: I think we have a typo here, right? (`Category Theory in Life`) We must fix the typo before merging the PR 😄

Comment (jorgegalindocruces) [reply README.md]: You are right! (I have a personal issue with the typos 😅)

**Comment by dreamingechoes** [comment]
Perfect! So, the only thing left is add yourself to the contributors' list, so everyone can know that you're part of the contributors to the repo! As we suggest on the contributions guidelines:

```
Feel free to add yourself to the contributors list (in alphabetical order) when you add a contribution. Use the following format: [Your Name](link). We want everyone to know the wonderful group of people who helped create this list.
```

😊

**Comment by dreamingechoes** [comment]
💖

**Pull request merged**